### PR TITLE
Unfied mode, enable!

### PR DESCRIPTION
**Pull request by damacus**
- unified-mode for Chef 17 compatability
- stop depending on selinux_policy
- stop depending on the ulimits cookbook

I plan to remove the ulimit config in a future release as I feel this is out of scope, and most people don't configure it. 

We should have an example of how to use it though!

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please provide a summary of your Pull Request.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by damacus** [comment]
There's a bug in the multi sentinel tests that is out of scope of this change. 

https://github.com/sous-chefs/redisio/issues/449

**Review by bmhughes — APPROVED** [review]

**Review by Xorima — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
 
 # redisio
 
+This file is used to list changes made in each version of the redisio cookbook.
+
 ## Unreleased
 
+- Set unified_mode true for Chef 17+ support
+- Require Chef 15.3+ for unified_mode
```
Comment (Xorima) [inline CHANGELOG.md:9]: wording here feels wrong as we are going to 16+ due to user_ulimit

Comment (damacus) [reply CHANGELOG.md:9]: I split it out into two lines so I could call out both bumps.

**Comment by kitchen-porter** [comment]
Released as: [6.0.0](https://github.com/sous-chefs/redisio/releases/tag/6.0.0)

**Pull request merged**